.github/workflows/macos.yml

```diff
@@ -37,12 +37,12 @@ jobs:
       - name: package
         run: |
           # 拷贝依赖
-          macdeployqt bin/release/${targetName}.app -qmldir=. -verbose=1 -dmg
+          macdeployqt bin/release/${env:targetName}.app -qmldir=. -verbose=1 -dmg
       # 上传artifacts          
       - uses: actions/upload-artifact@v2
         with:
           name: ${targetName}_${{matrix.qt_ver}}.zip
-          path: bin/release/${targetName}.app
+          path: bin/release/${env:targetName}.app
```

Comment:
You should use `${{ env.targetName }}` to match the GHA YAML syntax ([Context and expression syntax for GitHub Actions - GitHub Docs § `env` context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#env-context)).